### ISSUE TITLE
Missed failed Scenario Outline test examples in the report

### ISSUE THREAD

**Issue by voleksa** [Invictum/serenity-reportportal-integration#185]
If Scenario Outline has multiple test examples, and any of those test examples fail - the next failed examples will not be reported to the Report Portal launch.
E.g of Scenario Outline:
```
 Scenario Outline: Test scenario with multiple examples
    Given I open the app
    When I do something <action>
    Then I assert that <action> is <assert>

    Examples:
      | action | assert |
      | test   | true   |
      | sleep  | false  |
      | eat    | true   |
      | work   | false  |
      | dammy  | false  |
      | walk   | true   |
```
After the execution, the result will be next:
```
  Examples:
      | action | assert | 
      | test   | true   | - passed, reported to RP
      | sleep  | false  | - failed, reported to RP
      | eat    | true   | - passed, reported to RP
      | work   | false  | - **failed, not be reported to RP**
      | dammy  | false  | - **failed, not be reported to RP**
      | walk   | true   | - passed, reported to RP
```
How it looks in IntelliJ IDEA
![image](https://github.com/user-attachments/assets/1c1a2521-bcd8-4c8e-8f0c-98aea63b7e08)
And how it is reported in RP, examples 4 and  5 were not added to the RP
![image](https://github.com/user-attachments/assets/a0bd0393-3de7-4d13-8cc4-84948e93f8ca)
I`m using the latest, 1.6.0, version of serenity-reportportal-integration

**Comment by grey-rain** [comment]
It appeared to be native Serenity behaviour, so current request is a "feature request", not a bug.
<img width="1044" alt="Screenshot 2024-08-30 at 08 36 17" src="https://github.com/user-attachments/assets/21aefcaa-8c33-4efa-8348-6cd93f4628cc">
Implementation should allow to take both paths - old one (native) and customised.

**Comment by grey-rain** [comment]
Closed by mistake.
Will leave it in a backlog as a feature request.

**Comment by grey-rain** [comment]
Fixed in 1.6.2
